doc/6/controllers/security/update-user-mapping/snippets/update-user-mapping.js

```diff
@@ -0,0 +1,20 @@
+try {
+  const response = await kuzzle.security.updateUserMapping({
+    properties: {
+      firstName: { type: 'text' },
```

Comment:
```suggestion
      firstName: { type: 'keyword' },
```

Reply:
The mapping is already set when creating users from previous tests. 
We cannot update the field type. 

If the `text` type is really a problem, maybe we can think of some other field name.

Reply:
Ok no problem, it was a nitpicking